cmake/modules/FindSmctemp.cmake

```diff
@@ -0,0 +1,45 @@
+#.rst:
+# FindSmctemp
+# -------
+# Finds the smctemp library
+#
+# This will define the following variables::
+#
+# SMCTEMP_FOUND - system has smctemp
+# SMCTEMP_INCLUDE_DIRS - the smctemp include directory
+# SMCTEMP_LIBRARIES - the smctemp libraries
+# SMCTEMP_DEFINITIONS - the smctemp definitions
+#
+# and the following imported targets::
+#
+#   SMCTEMP::SMCTEMP   - The smctemp library
+
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_SMCTEMP libsmctemp QUIET)
+endif()
+
+find_path(SMCTEMP_INCLUDE_DIR NAMES smctemp.h
+                           PATHS ${PC_SMCTEMP_INCLUDEDIR})
+find_library(SMCTEMP_LIBRARY NAMES smctemp
+                          PATHS ${PC_SMCTEMP_LIBDIR})
+
+set(SMCTEMP_VERSION ${PC_SMCTEMP_VERSION})
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(Smctemp
+                                  REQUIRED_VARS SMCTEMP_LIBRARY SMCTEMP_INCLUDE_DIR
+                                  VERSION_VAR SMCTEMP_VERSION)
+
+if(SMCTEMP_FOUND)
+  set(SMCTEMP_LIBRARIES ${SMCTEMP_LIBRARY})
+  set(SMCTEMP_INCLUDE_DIRS ${SMCTEMP_INCLUDE_DIR})
```

Comment:
Drop these two.

Look at https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindXSLT.cmake for a template. Encompass the file in The target check, and add the target name to the global property

Reply:
hopefully did this right :)